lib/core-server/src/utils/stories-json.ts

```diff
@@ -0,0 +1,65 @@
+import path from 'path';
+import fs from 'fs-extra';
+import glob from 'globby';
+import { logger } from '@storybook/node-logger';
+// import { Options } from '@storybook/core-common';
+import { readCsf } from '@storybook/csf-tools';
+
+interface ExtractedStory {
+  id: string;
+  kind: string;
+  name: string;
+  parameters: Record<string, any>;
+}
```

Comment:
Don't want to use our existing types for this? Last thing we need is yet-another-story type definition.